ksqldb-functional-tests/src/test/resources/query-validation-tests/json.json

```diff
@@ -1,31 +1,426 @@
+        {"topic": "OUTPUT", "key": null, "value": {"V": 0}}
+      ]
+    },
+    {
+      "name": "INT - value - anonymous - deserialize",
+      "statements": [
+        "CREATE STREAM INPUT (V INT) WITH (WRAP_SINGLE_VALUE=false, kafka_topic='input_topic', value_format='JSON');",
         "CREATE STREAM OUTPUT AS SELECT * FROM INPUT;"
       ],
       "inputs": [
         {"topic": "input_topic", "value": 10},
         {"topic": "input_topic", "value": null}
       ],
       "outputs": [
-        {"topic": "OUTPUT", "value": {"FOO": 10}},
+        {"topic": "OUTPUT", "value": {"V": 10}},
+        {"topic": "OUTPUT", "value": null}
+      ]
+    },
+    {
+      "name": "INT - value - anonymous - serialize",
+      "statements": [
+        "CREATE STREAM INPUT (V INT) WITH (kafka_topic='input_topic', value_format='JSON');",
+        "CREATE STREAM OUTPUT WITH (WRAP_SINGLE_VALUE=false) AS SELECT * FROM INPUT;"
+      ],
+      "inputs": [
+        {"topic": "input_topic", "value": {"V": 123}},
+        {"topic": "input_topic", "value": null}
+      ],
+      "outputs": [
+        {"topic": "OUTPUT", "value": 123},
+        {"topic": "OUTPUT", "value": null}
+      ]
+    },
+    {
+      "name": "BIGINT - key",
+      "statements": [
+        "CREATE STREAM INPUT (K BIGINT KEY, V INT) WITH (kafka_topic='input_topic', format='JSON');",
+        "CREATE STREAM OUTPUT AS SELECT * FROM INPUT;"
+      ],
+      "properties": {
+        "ksql.key.format.enabled": true
+      },
+      "inputs": [
+        {"topic": "input_topic", "key": 998877665544332211, "value": {"V": 0}},
+        {"topic": "input_topic", "key": null, "value": {"V": 0}}
+      ],
+      "outputs": [
+        {"topic": "OUTPUT", "key": 998877665544332211, "value": {"V": 0}},
+        {"topic": "OUTPUT", "key": null, "value": {"V": 0}}
+      ]
+    },
+    {
+      "name": "BIGINT - value - anonymous - deserialize",
+      "statements": [
+        "CREATE STREAM INPUT (V BIGINT) WITH (WRAP_SINGLE_VALUE=false, kafka_topic='input_topic', value_format='JSON');",
+        "CREATE STREAM OUTPUT AS SELECT * FROM INPUT;"
+      ],
+      "inputs": [
+        {"topic": "input_topic", "value": 10827272222},
+        {"topic": "input_topic", "value": null}
+      ],
+      "outputs": [
+        {"topic": "OUTPUT", "value": {"V": 10827272222}},
+        {"topic": "OUTPUT", "value": null}
+      ]
+    },
+    {
+      "name": "BIGINT - value - anonymous - serialize",
+      "statements": [
+        "CREATE STREAM INPUT (V BIGINT) WITH (kafka_topic='input_topic', value_format='JSON');",
+        "CREATE STREAM OUTPUT WITH (WRAP_SINGLE_VALUE=false) AS SELECT * FROM INPUT;"
+      ],
+      "inputs": [
+        {"topic": "input_topic", "value": {"V": 1234567890}},
+        {"topic": "input_topic", "value": null}
+      ],
+      "outputs": [
+        {"topic": "OUTPUT", "value": 1234567890},
+        {"topic": "OUTPUT", "value": null}
+      ]
+    },
+    {
+      "name": "DOUBLE - key",
```

Comment:
The PR description says this test is currently disabled. How so?

Reply:
Ah, I'll update the title.  I pushed another commit after creating the PR to enable the double testing.